mlir/include/mlir/Dialect/Vector/Transforms/LoweringPatterns.h

```diff
@@ -254,6 +254,16 @@ void populateVectorGatherLoweringPatterns(RewritePatternSet &patterns,
 void populateVectorMaskLoweringPatternsForSideEffectingOps(
     RewritePatternSet &patterns);
 
+/// Populate the pattern set with the following patterns:
+///
+/// [VectorMaskedLoadOpConverter]
+/// Turns vector.maskedload to vector.load + arith.select
+///
+/// [VectorMaskedStoreOpConverter]
+/// Turns vector.maskedstore to scf.for + scf.if + memref.store
+void populateVectorMaskedLoadStoreLoweringPatterns(RewritePatternSet &patterns,
```

Comment:
We should probably call this `populateVectorMaskedLoadStoreEmulationPatterns` or something like that and mention the emulation part in the doc. We can say that this is useful for targets without native vector masking support.